data_validation/__main__.py

```diff
@@ -131,11 +131,23 @@ def get_calculated_config(args, config_manager):
         col_list = None if args.hash == "*" else cli_tools.get_arg_list(args.hash)
         fields = config_manager._build_dependent_aliases("hash", col_list)
         aliases = [field["name"] for field in fields]
-
+        # config_manager.append_dependent_aliases(aliases)
         # Add to list of necessary columns for selective hashing in order to drop
         # excess columns with invalid data types (i.e structs) when generating source/target DFs
         if col_list:
```

Comment:
If a column list was provided, we should append the col_list _and_ the aliases to the dependent aliases. 
If not column list was provided, we don't need to populate dependent_aliases at all. 
So couldn't we keep the original code like so:
```
if col_list:
            config_manager.append_dependent_aliases(col_list)
            config_manager.append_dependent_aliases(aliases)
```